pool/type.gno

```diff
@@ -258,6 +258,8 @@ type Pool struct {
 	tickBitmaps TickBitmaps // maps tick index to tick bitmap
 
 	positions Positions // maps the key (caller, lower tick, upper tick) to a unique position
+
+	tickCrossHook func(poolPath string, tickId int32, zeroForOne bool)
```

Comment:
Maybe it's still wip ?? I really don't see any logic

Reply:
The logic is provided by staker/staker.gno#tickCrossHook